pkg/controllers/manageagent/manageagent.go

```diff
@@ -104,6 +104,7 @@ func (h *handler) getAgentBundle(ns string) ([]runtime.Object, error) {
 			Spec: fleet.BundleSpec{
 				BundleDeploymentOptions: fleet.BundleDeploymentOptions{
 					DefaultNamespace: h.systemNamespace,
+					Decrypt: true,
```

Comment:
We should allow for the user to set this.

Reply:
This can be resolved. @StrongMonkey: "We don't manage agent bundle in git. There won't be a chance that agent bundle is encrypted"